docs/topics/testing/advanced.txt

```diff
@@ -580,10 +580,15 @@ and tear down the test suite.
     If ``shuffle`` is an integer, test cases will be shuffled in a random order
     prior to execution, using the integer as a random seed. If ``shuffle`` is
     ``None``, the seed will be generated randomly. In both cases, the seed will
-    be logged to the console and set to ``self.shuffle_seed`` prior to running
-    tests. This option can be used to help detect tests that aren't properly
-    isolated. :ref:`Grouping by test class <order-of-tests>` is preserved when
-    using this option.
+    be logged and set to ``self.shuffle_seed`` prior to running tests. This
+    option can be used to help detect tests that aren't properly isolated.
+    :ref:`Grouping by test class <order-of-tests>` is preserved when using this
+    option.
+
+    ``logger`` can be used to pass a Python `Logger object`_. If provided, the
```

Comment:
Can use intersphinx to link to Python docs? 

E.g. in the logging docs we use :py:func:`logging.getLogger`.